conf/schema.xml

```diff
@@ -344,7 +345,6 @@
         unknown fields indexed and/or stored by default -->
    <field name="dc.description" type="text_fgs"  indexed="true"  stored="true" multiValued="true"/>
    <dynamicField name="*" type="text_fgs"  indexed="true"  stored="true" multiValued="true"/>
-<field name="_version_" type="long" indexed="true" stored="true"/>
```

Comment:
I believe this is necessary for the Solr cloud stuff? As in: An intentional divergence.